CollAction/Frontend/app/project/index.tsx

```diff
@@ -20,4 +20,13 @@ renderComponentIf(
 renderComponentIf(
   <FindProject controller={false} />,
   document.getElementById("projects-container")
-);
\ No newline at end of file
+);
+
+renderComponentIf(
+  <FindProject controller={false} projectId={getQueryString("projectId") } />,
+  document.getElementById("embedded-projects-container")
+);
+
+function getQueryString(key) {
+  return decodeURIComponent(window.location.search.replace(new RegExp("^(?:.*[&\\?]" + encodeURIComponent(key).replace(/[\.\+\*]/g, "\\$&") + "(?:\\=([^&]*))?)?.*$", "i"), "$1"));
```

Comment:
This seems a .. very complicated way of doing things. Are you sure this can't be simplified a bit?

Reply:
There is; URLSearchParams. Unfortunately, not supported by all browsers (https://caniuse.com/#search=URLSearchParams). Got this one from StackOverflow.